src/components/FarmCard/FarmCardDetails.tsx

```diff
@@ -405,193 +500,187 @@ const FarmCardDetails: React.FC<{
               </Box>
               <Box
                 className={
-                  !approving &&
-                  !attemptStaking &&
-                  Number(stakeAmount) > 0 &&
-                  Number(stakeAmount) <=
-                    getExactTokenAmount(userLiquidityUnstaked)
+                  !attemptUnstaking &&
```

Comment:
Consolidate with other func in comment created above

Reply:
I did not update this part as well.